docs/help.md

```diff
@@ -21,4 +21,4 @@ If you want to report a bug:
 - be sure to first read about [how to contribute](https://github.com/docker/distribution/blob/master/CONTRIBUTING.md)
 - you can then do so on the [GitHub project bugtracker](https://github.com/docker/distribution/issues)
 
-You can also find out more about the Docker's project [Getting Help resources](https://docs.docker.com/opensource/get-help/).
+You can also find out more about the Docker's project [Getting Help resources](/opensource/get-help.md).
```

Comment:
we definitely need to work on this - the norm atm, is to link to the .md file, and work towards making the rendered version useful enough that everyone prefers to read there.